README.md

```diff
@@ -20,11 +20,32 @@ There are several options to integrate Mockup.
     <script src="https://cdn.jsdelivr.net/npm/@plone/mockup@latest/dist/bundle.min.js"></script>
     ```
 
+## Prerequisites
 
-## Install
+Prepare your system by installing prerequisites.
+
+### pre-commit
+
```

Comment:
I am a big fan of not copy-pasting docs, and instead referring to an authoritative source that is actively maintained. This reduces maintenance burden, but you would have to trust that the authoritative source will be around. In this case, I don't see `plone.api` going away anytime soon. See subsequent comment for an example of what I mean.

Reply:
Per discussion in Discord, pre-commit is not necessary after all, but I still like a Prerequisites section for the rest of the stuff.

Sorry for the noise. At least you have it installed on your system for when other Plone projects require it, which is quite a few due to `plone/meta`.